firefox-ios/Client/metrics.yaml

```diff
@@ -4260,7 +4314,7 @@ urlbar:
     bugs:
       - https://mozilla-hub.atlassian.net/browse/FXIOS-8111
     data_reviews:
-      - https://github.com/mozilla-mobile/firefox-ios/pull/18654
+      - https://github.com/mozilla-mobile/firefox-ios/pull/18507
```

Comment:
These data reviews should remain. So you shouldn't update, but add a new line for this PR.

Reply:
updated this as the merge deleted the whole last part of the event